tensorrt_llm/_torch/models/modeling_deepseekv3.py

```
@@ -732,15 +734,60 @@ def forward(
             hidden_states = self.input_layernorm(hidden_states)
+        
         # Self Attention
-        hidden_states = self.self_attn(
-            position_ids=position_ids,
-            hidden_states=hidden_states,
-            attn_metadata=attn_metadata,
-            all_reduce_params=AllReduceParams(
-                enable_allreduce=not (self.disable_attn_allreduce)),
-            **kwargs,
-        )
+        print(f"[DEBUG] DeepseekV3DecoderLayer.forward - position_ids shape: {position_ids.shape if position_ids is not None else None}")
```

Ruff (0.12.2)

Line too long (137 > 120)

(E501)